### PR TITLE
fix a typo and add home link

### PR DESCRIPTION
**Pull request by jonyhy96**
Fix a typo in Readme.md line 37
Change tables->table
Add home link in Chart.yaml
```release-note
None
```

### CODE REVIEW

File: stable/prometheus-postgres-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ spec:
                 secretKeyRef:
                   key: data_source_name
                   name: {{ template "prometheus-postgres-exporter.fullname" . }}
+          args: ["-extend.query-path", "/config"]
```
Comment (timstoop) [inline stable/prometheus-postgres-exporter/templates/deployment.yaml]: Seems like the exporter doesn't actually know this argument:
```
I0404 14:27:46.285] postgres_exporter: error: unexpected extend.query-path=/config, try --help
```

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by jonyhy96** [comment]
Can't fix pull-charts-e2e — Job failed promblem 
close it

**Pull request closed without merging**